packages/core/src/di/plugin-loader.ts

```diff
@@ -63,8 +63,8 @@ export class PluginLoader implements PluginResolver {
       if (typeof pluginExpression === 'string') {
         if (pluginExpression.includes('*')) {
           // Plugin directory is the node_modules folder of the module that installed stryker
-          // So if current __dirname is './@stryker-mutator/core/src/di' so 4 directories above
-          const pluginDirectory = path.dirname(path.resolve(__dirname, '..', '..', '..', '..', pluginExpression));
+          // So if current __dirname is './@stryker-mutator/core/dist/src/di' so 4 directories above
+          const pluginDirectory = path.dirname(path.resolve(__dirname, '..', '..', '..', '..', '..', pluginExpression));
```

Comment:
even more dots 😓

Reply:
haha yes. No way around that I fear. I don't want to rely on `process.cwd` because one could run Stryker from a different directory. The only other thing we can use is `__dirname` and this is a deep file.

Luckily our e2e tests cover this, so it is tested.